python/ray/dag/input_node.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, *args, _other_args_to_resolve=None, **kwargs):
         if len(args) != 0 or len(kwargs) != 0:
             raise ValueError("InputNode should not take any args or kwargs.")
 
+        self.input_nodes = {}
```

Comment:
Is there another variable name we can use for this? It's a bit strange to have the `InputNode` object store a reference to what seems like multiple `input_nodes`.

Reply:
Renamed to `input_attribute_nodes`. Lmk if there are other suggestions!